sdk/identity/Azure.Identity/src/AzurePowerShellCredential.cs

```diff
@@ -185,7 +189,7 @@ private static void CheckForErrors(string output)
                 WorkingDirectory = DefaultWorkingDir
             };
 
-        private void GetFileNameAndArguments(string resource, out string fileName, out string argument)
+        private void GetFileNameAndArguments(string resource, string tenantId, out string fileName, out string argument)
```

Comment:
Similar to my comment on `AzureCliCredential` should `TenantId` also be configurable when we construct the `AzurePowerShellCredential`?